script/model/type.py

```diff
@@ -54,6 +39,51 @@ class OpUnit(enum.IntEnum):
     UPDATE = 24,
     DELETE = 25,
     CREATE_INDEX = 26,
+    CREATE_INDEX_MAIN = 27,
+    PARALLEL_MERGE_HASHJOIN = 28,
+    PARALLEL_MERGE_AGGBUILD = 29,
+    PARALLEL_SORT_STEP = 30,
+    PARALLEL_SORT_MERGE_STEP = 31
+
+
+class ExecutionFeature(enum.IntEnum):
+    # Debugging information
+    QUERY_ID = 0,
+    PIPELINE_ID = 1,
+
+    # # features
+    NUM_FEATURES = 2,
+    FEATURES = 3,
+
+    # input features
+    EXEC_MODE = 4,
+    EST_OUTPUT_ROWS = 5,
```

Comment:
Oh another minor thing, can you rename this to just `NUM_ROWS`? Since it really represents the number of rows processed but not the number of output rows. I guess you need to modify the c++ side as well.